src/SourceBuild/content/eng/tools/tasks/Microsoft.DotNet.SourceBuild.Tasks.XPlat/UpdateNuGetConfigPackageSourcesMappings.cs

```diff
@@ -54,6 +54,8 @@ public class UpdateNuGetConfigPackageSourcesMappings : Task
         private Dictionary<string, List<string>> previouslyBuiltPackages = [];
         private Dictionary<string, List<string>> oldSourceMappingPatterns = [];
 
+        private string[] CustomSources = ["prebuilt", "net-sdk-supporting-feed"];
```

Comment:
```suggestion
        private readonly string[] CustomSources = ["prebuilt", "net-sdk-supporting-feed"];
```

Reply:
I question if prebuilt should be handled the same as net-sdk-supporting-feed.  The prebuilt feed is a local feed which can be enumerated and have full packageIds added for each of it's packages.  It should not have the cumulative list of PackageIDs added.

Reply:
Yeah, makes sense. It seemed that we should cover it when we discussed it yesterday, but it does not seem correct.

Reply:
Please open an issue to track this.